CHANGELOG.md

```diff
@@ -97,20 +99,87 @@
 **ATTENTION**: Our Doxygen documentation (e.g. [Storage
 docs][storage-dox-link]) was incorrectly showing the versioned inline namespace
 name for our symbols (it was `v1`), implicitly suggesting that users should
-spell this inline namespace in their own code. This mistake was fixed and our
-Doxygen documentation no longer shows the versioned inline namespace name, now
-showing users how to correctly spell our symbol names without referencing the
-versioned inline namespace.
+spell this inline namespace in their own code. This has been corrected. Our
+Doxygen documentation no longer shows the versioned inline namespace name;
+instead, it shows users how to correctly spell our symbol names without
+referencing the versioned inline namespace.
 
 [storage-dox-link]: https://googleapis.dev/cpp/google-cloud-storage/latest/
 
+### [BigQuery](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/bigquery/README.md)
+
+**BREAKING CHANGE**
```

Comment:
This is not breaking, we defined `gcpcxxV1` as a backwards compatibility alias.  The break will be in 1 year from now.